src/wrappers/themis/react-native-themis/src/index.js

```diff
@@ -88,12 +80,8 @@ export function secureCellSealWithSymmetricKeyDecrypt64(symmetricKey64, encrypte
 }
 ;
 export function secureCellSealWithPassphraseEncrypt64(passphrase, plaintext, context = "") {
-    if (passphrase === "" || passphrase === undefined || passphrase === null) {
-        throw new Error("Parameter passphrase can not be empty");
-    }
-    if (plaintext === "" || plaintext === undefined || plaintext === null) {
-        throw new Error("Parameter plaintext can not be empty");
-    }
+    convertInputBase64(plaintext, "plaintext", false);
+    convertInputBase64(passphrase, "passphrase", false);
```

Comment:
It looks weird to call a function that "converts" something, and don't do anything with the result. You can reassign it back to the same variable, for example. Or name the non-converting variant something like `checkInput` that does not return anything, only throws errors.

Reply:
done